File: Part8/C++/FunctionConcepts/FunctionConcepts_14.cpp
// Main function with parameters to access command line arguments
#include <iostream>
#include <cstdio>
using namespace std;

int main ( int argc, char* argv[] )
{
    printf ( "Number of arguments: %d\n", argc );
    printf ( "Arguments:\n" );
    for ( int i = 0; i < argc; i++ )
    {
        printf ( "\tArgument %d: %s\n", i, argv[i] );
    }
    return 0;
}

File: Part8/C#/FunctionConcepts/FunctionConcepts_14.cs
// Main method with parameters to access command line arguments
using System;

class Program
{
    static void Main ( string[] args )
    {
        Console.WriteLine ( "Number of arguments: {0}", args.Length );
        Console.WriteLine ( "Arguments:" );
        for ( int i = 0; i < args.Length; i++ )
        {
            Console.WriteLine ( "\tArgument {0}: {1}", i, args[i] );
        }
    }
}
